deploy/004_deploy_tokens.ts

```diff
@@ -0,0 +1,52 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { ethers } from "hardhat";
+
+module.exports = async function (hre: HardhatRuntimeEnvironment) { // HardhatRuntimeEnvironment
+  const {deployments, network, getNamedAccounts} = hre;
+  const { log } = deployments;
+  const [deployer] = await ethers.getSigners();
+  const { kovanSideToken, binanceSideToken } = await getNamedAccounts();
+
+  const SwapRBTC = await deployments.get('SwapRBTC');
+  const SwapRrbtcProxy = await deployments.get('SwapRbtcProxy');
+  const swapContract = new ethers.Contract(SwapRrbtcProxy.address, SwapRBTC.abi, deployer);
+  
+  if(!network.live){
+    return;
+  }
+
+  if(kovanSideToken){
+    const containsSideToken = await swapContract.containsSideTokenBtc(kovanSideToken);
+    if(containsSideToken){
+      log('Token with address: ', kovanSideToken, ' already in contract.')
+    }
+    else {
+      await swapContract.addSideTokenBtc(kovanSideToken);
+      if(await swapContract.containsSideTokenBtc(kovanSideToken)){
+        log('Added token with address: ', kovanSideToken)
+      }
+      else{
+        log('Could not add the token with address: ', kovanSideToken)
+      }  
+    } 
+  }
+
+  if(binanceSideToken){
+    const containsSideToken = await swapContract.containsSideTokenBtc(binanceSideToken);
```

Comment:
As it's the same code, i would refactor this to a function, it will be easier to add another sideTokenBtc that way in the future

Reply:
Yeah good idea. Refactored. Please review.